CHANGELOG.md

```diff
@@ -5,13 +5,41 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
-## [0.26.1] - 2022-03-10
+## [0.90.0] - 2022-03-14
```

Comment:
I specified v0.90.0 as a placeholder. My intention is to indicate that this is a significant change. Perhaps this is excessive, but I believe it would be effective.